src/Symfony/Component/Routing/Tests/Loader/AttributeClassLoaderTestCase.php

```diff
@@ -231,6 +240,23 @@ public function testLocalizedPrefixLocalizedRoute()
         $this->assertEquals('/en/action', $routes->get('action.en')->getPath());
     }
 
+    public function testLocalizedPassthroughLocaleRoute()
+    {
+        $routes = $this->loader->load(LocalizedPassthroughLocaleActionController::class);
+        $this->assertCount(4, $routes);
+        $this->assertEquals('/{_locale}', $routes->get('action')->getPath());
+        $this->assertEquals('/nl', $routes->get('action.nl')->getPath());
+        $this->assertEquals('/{_locale}', $routes->get('action2')->getPath());
+        $this->assertEquals('/nl', $routes->get('action2.nl')->getPath());
+    }
+
+    public function testLocalizedIndexLocaleRoute()
```

Comment:
looks like these two methods (`testLocalizedPassthroughLocaleRoute()` and `testLocalizedIndexLocaleRoute()`) accidentally made it in this PR

Reply:
Oops! That's what I get for trying to do two PR's at once. Thanks for the find!